amaranth/vendor/_gowin.py

```diff
@@ -440,10 +440,10 @@ def _osc_div(self):
         "{{name}}.sdc": r"""
```

Comment:
Would you mind adding a comment like this?
```suggestion
        # Gowin is using neither Tcl nor the Synopsys code to parse SDC files, so the grammar deviates from the usual.
        "{{name}}.sdc": r"""
```

Reply:
Sure. I made it into a separate commit, do you prefer it squashed in the initial PR commit?

Reply:
I would prefer squashed, thanks!